server/backend/readme.md

```diff
@@ -178,3 +178,27 @@ for test Output,
 
 - find the `output` tab
 - select `Python Test Log` in from the pulldown, top right of the output window
+
+
+## ----------------------------------------------------------------------------
+## -------------- Windows Configuration ----------------------------------------
+## ----------------------------------------------------------------------------
```

Comment:
Hi Ayobami, the header section looks a bit strange. Maybe it is ok to follow the one on top section?
![image](https://github.com/bcgov/nr-forests-access-management/assets/81595625/e074c78e-88e6-40ac-a530-6f69212ee9e7)